target/start-mailserver.sh

```diff
@@ -354,6 +354,14 @@ if [ "$ONE_DIR" = 1 -a -d $statedir ]; then
     fi
   done
 fi
+if [ "$ENABLE_ELK" = 1 ]; then
+ELK_PORT=${ELK_PORT:="10514"}
+ELK_HOST=${ELK_HOST:="elk"}
+echo "forward logs to ELK ($ELK_HOST:$ELK_PORT)"
+echo " *.* @$ELK_HOST:$ELK_PORT " > /etc/rsyslog.d/60-elk.conf
```

Comment:
i try use the resources that have mailserver image, but i will try add firebeat client